x-pack/filebeat/module/gsuite/admin/test/gsuite-admin-chromeos-test.json.log-expected.json

```diff
@@ -607,7 +607,6 @@
         ]
     },
     {
-        "@timestamp": "2020-10-02T15:00:00.000Z",
```

Comment:
seems odd that this is removed.  Looks like the date is in the original log.

Reply:
It must be from https://github.com/elastic/beats/blob/3390aa84ce88c6c62aeca97ac1cc0b5485fdceb0/filebeat/tests/system/test_modules.py#L226-L257